src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/SdkModels/Resources/PSResource.cs

```diff
@@ -62,7 +65,7 @@ public PSResource(GenericResource resource)
             this.ManagedBy = resource.ManagedBy;
             this.Name = resource.Name;
             this.Plan = resource.Plan;
-            this.Properties = resource.Properties;
+            this.Properties = ((JToken)resource.Properties).ToPsObject();
```

Comment:
Nice.

Also, argh PowerShell, haha.

Reply:
Also, is it ever possible for `resource.Properties` to not be convertible to `JToken`?